script/release-docker

```diff
@@ -16,35 +16,68 @@
 #
 ################################################################################
 #
+
+# Build the docker images, and upload them to gcloud.
+#
+# For development, you can pass "debug" parameter to only build debug image and skip the upload.
+# You can override DEBUG_IMAGE_NAME, RELEASE_IMAGE_NAME to specify a different destination for upload.
+# You can also set TAG, PROJECT, REPO - the upload will go to $REPO/$PROJECT/envoy[-debug]:$TAG
```

Comment:
can you copy the code from pilot/bin/e2e.sh (-hub -tag etc. ?) So that we have consistency across repos. And its more flexible here (no env vars